arch/arm/configs/mako_defconfig

```diff
@@ -530,3 +530,23 @@ CONFIG_FANOTIFY=y
 CONFIG_ANDROID_PARANOID_NETWORK=n
 CONFIG_RTC_DRV_CMOS=y
 CONFIG_ANDROID_LOW_MEMORY_KILLER=n
+CONFIG_CGROUP_MEM_RES_CTLR=y
```

Comment:
This should be commented out to be able to boot Sailfish OS update10 or newer, investigating why (kernel boot panics, can't see reason as not got UART wired up :) How is NemoMobile looking with it?